ihtml/themes/breezy/login.tpl

```diff
@@ -76,6 +75,10 @@
 
 </div>
 
+<div class="copynotice">
+  &copy; 2002-{$year} <a href="http://www.fusionDirectory.org">{t}The FusionDirectory team{/t}, {$revision}</a>
+</div>
+
```

Comment:
Maybe this should go into one big translatable string to ease translation for RTL languages.
So: `{t 1=$year 2=$revision}&copy; 2002-%1 <a href="http://www.fusiondirectory.org">The FusionDirectory team, %2</a>{/t}`